install/get-tor-osx.py

```diff
@@ -37,7 +37,7 @@
 def main():
     dmg_url = 'https://archive.torproject.org/tor-package-archive/torbrowser/7.5.5/TorBrowser-7.5.5-osx64_en-US.dmg'
     dmg_filename = 'TorBrowser-7.5.5-osx64_en-US.dmg'
```

Comment:
@micahflee the version of the .dmg here is still 7.5.5, was it meant to be updated to 8.0 like in `get-tor-windows.py`?

Reply:
Oh hah good review. I tested the actual packaging in Windows, but didn't test the actual packaging in OSX